WordPress/Classes/ViewRelated/Jetpack/Branding/Overlay/JetpackFullscreenOverlayGeneralPhaseConfig.swift

```diff
@@ -0,0 +1,221 @@
+            static let statsContinueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.stats.continue.title",
+                                                                    value: "Continue to Stats",
+                                                                    comment: "Title of a button that dismisses an overlay and displays the Stats screen.")
+            static let readerContinueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.reader.continue.title",
+                                                                     value: "Continue to Reader",
+                                                                     comment: "Title of a button that dismisses an overlay and displays the Reader screen.")
+            static let notificationsContinueButtonTitle = NSLocalizedString("jetpack.fullscreen.overlay.notifications.continue.title",
+                                                                            value: "Continue to Notifications",
+                                                                            comment: "Title of a button that dismisses an overlay and displays the Notifications screen.")
+        }
+
+        enum PhaseOne {
+
+            enum Stats {
+                static let title = NSLocalizedString("jetpack.fullscreen.overlay.phaseOne.stats.title",
+                                                     value: "Get your stats using the new Jetpack app",
+                                                     comment: "Title of a screen displayed when the user accesses the Stats screen from the WordPress app. The screen showcases the Jetpack app.")
+                static let subtitle = NSLocalizedString("jetpack.fullscreen.overlay.phaseOne.stats.subtitle",
+                                                     value: "Switch to the Jetpack app to watch your site’s traffic grow with stats and insights.",
+                                                     comment: "Subtitle of a screen displayed when the user accesses the Stats screen from the WordPress app. The screen showcases the Jetpack app.")
+            }
+
+            enum Reader {
+                static let title = NSLocalizedString("jetpack.fullscreen.overlay.phaseOne.reader.title",
+                                                     value: "Follow any site with the Jetpack app",
+                                                     comment: "Title of a screen displayed when the user accesses the Reader screen from the WordPress app. The screen showcases the Jetpack app.")
+                static let subtitle = NSLocalizedString("jetpack.fullscreen.overlay.phaseOne.reader.subtitle",
+                                                     value: "Switch to the Jetpack app to find, fillow, and like all your favorite sites and posts with Reader.",
```

Comment:
[nit] typo
```suggestion
                                                     value: "Switch to the Jetpack app to find, follow, and like all your favorite sites and posts with Reader.",
```

Reply:
Thanks for catching this! Fixed in e3139826b55785ab1cca29cfda1330a2b58df399 👍